packages/smooth_app/lib/pages/product/add_new_product_page.dart

```diff
@@ -94,31 +94,29 @@ class _AddNewProductPageState extends State<AddNewProductPage> {
                 ),
                 Positioned(
                   child: Align(
-                    alignment: Alignment.bottomCenter,
-                    child: SmoothActionButtonsBar.single(
-                      action: SmoothActionButton(
-                        text: appLocalizations.finish,
-                        onPressed: () async {
-                          final LocalDatabase localDatabase =
-                              context.read<LocalDatabase>();
-                          final DaoProduct daoProduct =
-                              DaoProduct(localDatabase);
-                          final Product? localProduct =
-                              await daoProduct.get(widget.barcode);
-                          if (localProduct == null) {
-                            product = Product(
-                              barcode: widget.barcode,
-                            );
-                            daoProduct.put(product);
-                            localDatabase.notifyListeners();
-                          }
-                          provider.set(product);
-                          if (mounted) {
-                            await Navigator.maybePop(context,
-                                _isProductLoaded ? widget.barcode : null);
-                          }
-                        },
-                      ),
+                    alignment: Alignment.bottomRight,
+                    child: FloatingActionButton.extended(
```

Comment:
Forget about `Positioned` and `Align`: just use the `floatingActionButton` parameter of `SmoothScaffold`.

Reply:
This type of behaviour SmoothScaffold is showing.

<img src="https://user-images.githubusercontent.com/75238158/198046716-6dc6d468-5350-49cd-b3c7-f82083433232.jpg" width="250" height="500">

Reply:
This looks like similar to your initial screenshot, isn't-it?

Reply:
Yes, it is similar but SmoothScaffold is hiding other contents

Reply:
> Yes, it is similar but SmoothScaffold is hiding other contents on screen

Reply:
Here I found something to display other screen stuff by using the background colour property in SmoothScaffold as transparent

![Screenshot 2022-10-26 at 7 48 56 PM](https://user-images.githubusercontent.com/75238158/198051205-7f7d1633-230a-490e-b685-c53a21cc7b44.png)

Reply:
@monsieurtanuki @g123k should I commit this code?

Reply:
@prathamsoni11 It looks like you removed the `body` from the `SmoothScaffold`: that would explain the empty screen.

Reply:
@monsieurtanuki @g123k  Done it is now ready to be merged.